content/blog/2017/08/2017-08-10-kubernetes-with-pipeline-acs.adoc

```diff
@@ -118,7 +118,7 @@ Azure on Jenkins, and vice versa, such as:
 * **Hosted Jenkins**. New
 link:https://azuremarketplace.microsoft.com/en-us/marketplace/apps/azure-oss.jenkins[Solution
 Template] in Azure Marketplace lets you spin up a
-Jenkins Master on Azure in minutes. Not only is it easy and fast, the solution
+Jenkins Controller on Azure in minutes. Not only is it easy and fast, the solution
```

Comment:
Unclear why uppercase

Reply:
I left the original case type the same, so if it was upper case I left it upper case